pkg/api/v1alpha1/tinkerbellmachineconfig.go

```diff
@@ -37,128 +30,14 @@ func NewTinkerbellMachineConfigGenerate(name string) (*TinkerbellMachineConfigGe
 				Name:              "ec2-user",
```

Comment:
the indentation here has gone funky because of the double `{{` which, at a glance, makes it look like `Name` and `SshAuhotirzedKeys` are part of `TinkerbellMachineConfigSpec`. Would you mind re-formatting so the indentation helps glance-ability.

Reply:
Nice catch! I did not notice that.